CHANGELOG.rst

```diff
@@ -1,5 +1,9 @@
-Will be updated by antsibull-changelog. Do not edit this manually!
```

Comment:
This file gets generated by `antsibull-changelog` from changelog fragments. No need to revert the change, just a heads-up that it will get overwritten (assuming we use that release process).

Reply:
I think that delete was via antsibull - I did a 0.0.0 release just to test it out, see lines below.